plugins/CorePluginsAdmin/PluginInstaller.php

```diff
@@ -297,12 +297,19 @@ private function fixPluginFolderIfNeeded($tmpPluginFolder)
 
     private function copyPluginToDestination($tmpPluginFolder)
     {
-        $pluginsDir = Manager::getPluginsDirectory();
-        $pluginTargetPath = $pluginsDir . $this->pluginName;
+        // if MATOMO_PLUGIN_COPY_DIR is defined , copy plugin to this directory too
 
-        $this->removeFolderIfExists($pluginTargetPath);
+        $pluginsDirs = [Manager::getPluginsDirectory()];
+        
+        if (!empty($GLOBALS['MATOMO_PLUGIN_COPY_DIR'])) {
```

Comment:
Not sure why you are copying the plugin to two directories? Shouldn't it only copy the plugin to one directory?

Reply:
I thought it's important to copy plugin to Matomo's /plugins directory too.
I changed the code to use only one directory